R/jobs.R

```diff
@@ -8,37 +8,26 @@
 #'
 #' @param file File to be used as entrypoint for training.
 #'
-#' @param config The name of the configuration to be used. Defaults to the
-#'   `"cloudml"` configuration.
-#'
 #' @param scale_tier Machine type for training. "basic" provides a single
 #'   worker instance suitable for learning how to use Cloud ML Engine,
 #'   and for experimenting with new models using small datasets; "basic-gpu"
 #'   provides a single worker instance with a GPU. "basic-tpu" provides a
 #'   single worker instance with a TPU.
 #'
-#' @param hypertune
-#'   Path to a YAML file, defining how hyperparameters should
-#'   be tuned. See
-#'   https://cloud.google.com/ml/reference/rest/v1/projects.jobs
-#'   for more details.
-#'
-#' @param collect Collect job output after submission. Defaults to "ask" which
-#'   will prompt within interactive environments. Within versions of RStudio
-#'   that support terminals (>= v1.1), the waiting and collection will be done
-#'   within a terminal.
+#' @param cloudml A list, \code{YAML} or \code{JSON} configuration file as described
+#'   \url{https://cloud.google.com/ml-engine/reference/rest/v1/projects.jobs}.
 #'
-#' @param ... Named arguments, used to supply runtime configuration settings to
-#'   your TensorFlow application.
+#' @param gcloud A list or \code{YAML} file with optional 'account' or 'project'
+#'   fields used to configure the GCloud environemnt.
```

Comment:
Don't forget the `configuration` (and possibly other fields). Recall that `configuration` was for named configurations (i.e. account/project profiles)

Reply:
I think we technically might need the `gcloud` argument on all of the methods that interact with `gcloud` (as otherwise they end up using the global default)